pom.xml

```diff
@@ -13,6 +13,12 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <jenkins-test-harness.version>1.625</jenkins-test-harness.version>
```

Comment:
I'm ok with bumping the baseline to `1.625`. `1.424` is just too old. But updating the JTH is really needed?